### PR TITLE
(feat-Dartsmiths): add login page

### PR DESCRIPTION
**Pull request by lealemgetahun**
I implemented a login page with a login button, a2sv logo, and a text form field. the input form accepts the username and password. there is also a reset password option for users who may have forgotten their password.

the screenshot was taken before the review 

to be pushed

screenshot take after review  
new_screen

### CODE REVIEW

**Review by BekyH — COMMENTED** [review]
Attach the screenshot

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import 'package:dartsmiths/features/login/presentation/widgets/login_card.dart';
+import 'package:flutter/material.dart';
+
+class Login extends StatelessWidget {
+  const Login({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white,
```
Comment (BekyH) [inline Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart:11]: Avoid defining colors like this. rebase the main, define the colors in the color constant file and use them.

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+              image: AssetImage('assets/images/a2sv_logo.jpg'),
+            ),
+          ),
+          const SizedBox(
+            height: 10,
+          ),
+          Expanded(
+            child: Container(
+              decoration: const BoxDecoration(
+                color: Color(0xFF376AED),
```
Comment (BekyH) [inline Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart:26]: color the same thing with above.

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/framework.dart';
+import 'package:flutter/src/widgets/placeholder.dart';
+
+// ignore: camel_case_types
+class login_card extends StatelessWidget {
```
Comment (BekyH) [inline Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart:6]: Use UpperCamelCase for class names.

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+                      ),
+                    ),
+                  ),
+                ),
+                const SizedBox(
+                  height: 50,
+                ),
+                SizedBox(
+                  height: 60,
+                  width: 295,
```
Comment (BekyH) [inline Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart:74]: The sizes are defined in the Ui converter which is inside the core/utils directory. make use of them.

**Review by BekyH — CHANGES_REQUESTED** [review]
Fix the above comments and also attach the screenshot.

**Review by Redi22 — CHANGES_REQUESTED** [review]
LGTM in general but some naming changes need to be made and also untrack the .dart_tool files as well.

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart (hunk 1 above)
Comment (Beki4382) [reply Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart]: I used color.dart file and update the colors used in the code

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart (hunk 2 above)
Comment (Beki4382) [reply Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart]: done

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart (hunk 3 above)
Comment (Beki4382) [reply Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart]: done

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart (hunk 4 above)
Comment (Beki4382) [reply Mobile/starter_project_dartsmiths/lib/features/login/presentation/widgets/login_card.dart]: I used UI converter file

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart
[the remark this answers is not in the document]
Comment (Beki4382) [reply Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart:1]: done

File: Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart [hunk 5]
```diff
@@ -0,0 +1,62 @@
+
+import 'package:dartsmiths/features/login/presentation/widgets/login_card.dart';
+import 'package:flutter/material.dart';
+
+class Login extends StatelessWidget {
```
[the remark this answers is not in the document]
Comment (Beki4382) [reply Mobile/starter_project_dartsmiths/lib/features/login/presentation/screen/Login_page.dart]: done

**Comment by Beki4382** [comment]
fix the login card and login page. I also implement custom login text to style the repeated login text.

**Review by Redi22 — APPROVED** [review]
LGTM

**Review by BekyH — APPROVED** [review]

**Pull request merged**